<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckList</class>
 <widget class="QMainWindow" name="CheckList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PCB_Design_CheckList</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_dateTime">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>所属项目：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_project"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>项目负责人：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_manager"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_checkList">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>22</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>23</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>24</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>25</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>26</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>27</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>28</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>29</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>31</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>32</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>33</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>34</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>35</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>36</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>37</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>38</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>39</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>40</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>41</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>42</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>43</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>44</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>45</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>46</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>47</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>48</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>项目</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>检查内容</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>yes（必填）</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>no（必填）</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>添加备注</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>schematic</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>确认所有器件封装是否正确.</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>母板与子板，单板与背板，确认信号对应，位置对应，连接器方向及丝印标识正确，</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>元器件是否100% 放置</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>打开器件TOP和BOTTOM层的place-bound， 查看重叠引起的DRC是否允许</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>Mark点是否足够且必要</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>较重的元器件，应该布放在靠近PCB支撑点或支撑边的地方，以减少PCB的翘曲</string>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>与结构相关的器件布好局后最好锁住，防止误操作移动位置</string>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string>压接插座周围5mm范围内，正面不允许有高度超过压接插座高度的元件，背面不允许有元件或焊点</string>
       </property>
      </item>
      <item row="8" column="1">
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item row="8" column="2">
       <property name="text">
        <string>确认器件布局是否满足工艺性要求（重点关注BGA、PLCC、贴片插座）</string>
       </property>
      </item>
      <item row="9" column="1">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="9" column="2">
       <property name="text">
        <string>金属壳体的元器件，特别注意不要与其它元器件相碰，要留有足够的空间位置</string>
       </property>
      </item>
      <item row="10" column="1">
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item row="10" column="2">
       <property name="text">
        <string>接口相关的器件尽量靠近接口放置，背板总线驱动器尽量靠近背板连接器放置</string>
       </property>
      </item>
      <item row="11" column="1">
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item row="11" column="2">
       <property name="text">
        <string>波峰焊面的CHIP器件是否已经转换成波峰焊封装，</string>
       </property>
      </item>
      <item row="12" column="1">
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item row="12" column="2">
       <property name="text">
        <string>手工焊点是否超过50个</string>
       </property>
      </item>
      <item row="13" column="1">
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item row="13" column="2">
       <property name="text">
        <string>在PCB上轴向插装较高的元件，应该考虑卧式安装。留出卧放空间。并且考虑固定方式，如晶振的固定焊盘</string>
       </property>
      </item>
      <item row="14" column="1">
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item row="14" column="2">
       <property name="text">
        <string>需要使用散热片的器件，确认与其它器件有足够间距，并且注意散热片范围内主要器件的高度</string>
       </property>
      </item>
      <item row="15" column="1">
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item row="15" column="2">
       <property name="text">
        <string>数模混合板的数字电路和模拟电路器件布局时是否已经分开，信号流是否合理</string>
       </property>
      </item>
      <item row="16" column="1">
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item row="16" column="2">
       <property name="text">
        <string>A/D转换器跨模数分区放置。</string>
       </property>
      </item>
      <item row="17" column="1">
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item row="17" column="2">
       <property name="text">
        <string>时钟器件布局是否合理</string>
       </property>
      </item>
      <item row="18" column="1">
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item row="18" column="2">
       <property name="text">
        <string>高速信号器件布局是否合理</string>
       </property>
      </item>
      <item row="19" column="1">
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item row="19" column="2">
       <property name="text">
        <string>端接器件是否已合理放置</string>
       </property>
      </item>
      <item row="20" column="0">
       <property name="text">
        <string>placement</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="20" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="20" column="2">
       <property name="text">
        <string>（源端匹配串阻应放在信号的驱动端；中间匹配的串阻放在中间位置；终端匹配串阻应放在信号的接收端）</string>
       </property>
      </item>
      <item row="21" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="21" column="2">
       <property name="text">
        <string>IC器件的去耦电容数量及位置是否合理</string>
       </property>
      </item>
      <item row="22" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="22" column="2">
       <property name="text">
        <string>信号线以不同电平的平面作为参考平面，当跨越平面分割区域时，参考平面间的连接电容是否靠近信号的走线区域</string>
       </property>
      </item>
      <item row="23" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="23" column="2">
       <property name="text">
        <string>保护电路的布局是否合理，是否利于分割</string>
       </property>
      </item>
      <item row="24" column="1">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="24" column="2">
       <property name="text">
        <string>单板电源的保险丝是否放置在连接器附近，且前面没有任何电路元件</string>
       </property>
      </item>
      <item row="25" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="25" column="2">
       <property name="text">
        <string>确认强信号与弱信号（功率相差30dB）电路分开布设</string>
       </property>
      </item>
      <item row="26" column="1">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="26" column="2">
       <property name="text">
        <string>是否按照设计指南或参考成功经验放置可能影响EMC实验的器件。如：面板的复位电路要稍靠近复位按钮</string>
       </property>
      </item>
      <item row="27" column="1">
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item row="27" column="2">
       <property name="text">
        <string>对热敏感的元件（含液态介质电容、晶振）尽量远离大功率的元器件、散热器等热源</string>
       </property>
      </item>
      <item row="28" column="1">
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item row="28" column="2">
       <property name="text">
        <string>布局是否满足热设计要求，散热通道（根据工艺设计文件来执行）</string>
       </property>
      </item>
      <item row="29" column="1">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="29" column="2">
       <property name="text">
        <string>是否IC电源距离IC过远</string>
       </property>
      </item>
      <item row="30" column="1">
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item row="30" column="2">
       <property name="text">
        <string>LDO及周围电路布局是否合理</string>
       </property>
      </item>
      <item row="31" column="1">
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item row="31" column="2">
       <property name="text">
        <string>模块电源等周围电路布局是否合理</string>
       </property>
      </item>
      <item row="32" column="1">
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item row="32" column="2">
       <property name="text">
        <string>电源的整体布局是否合理</string>
       </property>
      </item>
      <item row="33" column="1">
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item row="33" column="2">
       <property name="text">
        <string>是否所有仿真约束都已经正确加到Constraint Manager中</string>
       </property>
      </item>
      <item row="34" column="1">
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item row="34" column="2">
       <property name="text">
        <string>是否正确设置物理和电气规则（注意电源网络和地网络的约束设置）</string>
       </property>
      </item>
      <item row="35" column="1">
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item row="35" column="2">
       <property name="text">
        <string>Test Via、Test Pin的间距设置是否足够</string>
       </property>
      </item>
      <item row="36" column="1">
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item row="36" column="2">
       <property name="text">
        <string>叠层的厚度和方案是否满足设计和加工要求</string>
       </property>
      </item>
      <item row="37" column="1">
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item row="37" column="2">
       <property name="text">
        <string>所有有特性阻抗要求的差分线阻抗是否已经经过计算，并用规则控制</string>
       </property>
      </item>
      <item row="38" column="0">
       <property name="text">
        <string>layout</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="38" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="38" column="2">
       <property name="text">
        <string>是否IC电源距离IC过远</string>
       </property>
      </item>
      <item row="39" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="39" column="2">
       <property name="text">
        <string>LDO及周围电路布局是否合理</string>
       </property>
      </item>
      <item row="40" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="40" column="2">
       <property name="text">
        <string>模块电源等周围电路布局是否合理</string>
       </property>
      </item>
      <item row="41" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="41" column="2">
       <property name="text">
        <string>电源的整体布局是否合理</string>
       </property>
      </item>
      <item row="42" column="0">
       <property name="text">
        <string>dfm</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="42" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="42" column="2">
       <property name="text">
        <string>器件位号是否遗漏，位置是否能正确标识器件</string>
       </property>
      </item>
      <item row="43" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="43" column="2">
       <property name="text">
        <string>器件位号命名是否准确，按照分类否符合要求</string>
       </property>
      </item>
      <item row="44" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="44" column="2">
       <property name="text">
        <string>确认器件的管脚排列顺序， 第1脚标志，器件的极性标志，连接器的方向标识的正确性</string>
       </property>
      </item>
      <item row="45" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="45" column="2">
       <property name="text">
        <string>背板是否正确标识了槽位名、槽位号、端口名称、护套方向</string>
       </property>
      </item>
      <item row="46" column="1">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="46" column="2">
       <property name="text">
        <string>确认已经放置有防静电和射频板标识（射频板使用）</string>
       </property>
      </item>
      <item row="47" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="47" column="2">
       <property name="text">
        <string>确认PCB编码正确，且根据自身公司情况，设置PCB编码规范。</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
